lnbits/static/js/base.js

```diff
@@ -144,12 +144,15 @@ window.LNbits = {
           id: wallet_id
         })
         .then(_ => {
+          credit = parseInt(credit)
           Quasar.Notify.create({
```

Comment:
this notify should not be in base.js, this function just should return the promise